### PR TITLE
I worked on the Submit Page using HTML

### PR DESCRIPTION
**Pull request by nwakaemecynthia**
## `I worked on the Submit Page using HTML`
nav
submit1

### CODE REVIEW

**Review by michaelolubajo — COMMENTED** [review]
Good job! I like that it goes straight to the point.
For the logo we'll need to link to a single file in an images folder for all 3 pages. 
Also, our nav bars need to have a uniform look. By the way, the image you shared shows a duplication of the links at the top.
We're almost there.
Well done

**Comment by nwakaemecynthia** [comment]
@michaelolubajo I added you as a reviewer. You can approve so Yemi can Merge the post.
Hopeful by the end of today we will get other Team members' work and effect changes where necessary.

Thank You.

**Review by michaelolubajo — APPROVED** [review]

File: Covid-watch-by-cynthia/submit.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <div>
+      <form action="#" method="post">
+        <fieldset>
+          <label for="blog-title">Title</label>
+          <br>
+          <input type="text" name="blog-title" id="blog-titile">
+          <br><br>
+          <label for="blog-details">News</label>
+          <br>
+          <textarea name="" id="" cols="100" rows="20"></textarea>
```
Comment (yemexx1) [inline Covid-watch-by-cynthia/submit.html]: you're missing the name here, could you add it? 
```suggestion
          <textarea name="blog-details" id="" cols="100" rows="20"></textarea>
```

File: Covid-watch-by-cynthia/submit.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Submit</title>
```
Comment (yemexx1) [inline Covid-watch-by-cynthia/submit.html]: Let's make this full - `Submit News`
```suggestion
  <title>Submit News</title>
```

File: Covid-watch-by-cynthia/submit.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          <br><br>
+          <label for="blog-details">News</label>
+          <br>
+          <textarea name="" id="" cols="100" rows="20"></textarea>
+          <br><br>
+          <label for="image-upload">Image</label>
+          <br>
+          <input type="file" name="image-upload" id="image-upload">
+          <br><br>
+          <button type="submit">SUBMIT</button>
```
Comment (yemexx1) [inline Covid-watch-by-cynthia/submit.html]: Let's make submit also title case here.
```suggestion
          <button type="submit">Submit</button>
```

**Review by yemexx1 — CHANGES_REQUESTED** [review]
Nice work! Please take a look at inline comments.

Like I mentioned in #2, we can put the html files in the root of the project.

**Review by yemexx1 — CHANGES_REQUESTED** [review]
@Cynthia-Berry could you move the image into images folder to align with master?

**Comment by nwakaemecynthia** [comment]
> @Cynthia-Berry could you move the image into images folder to align with master?

I ran a git fetch but couldn't get Michael's image folder. That's why I have a  `img`  in my PR

**Review by michaelolubajo — APPROVED** [review]

**Review by yemexx1 — APPROVED** [review]

**Pull request merged**